lib/common/idioms.h

```diff
@@ -133,6 +133,10 @@ template<typename A> struct ListItemCount {
 // Given a const reference to a value, return a copy of the value.
 template<typename A> A Clone(const A &x) { return x; }
 
+template<typename A> inline bool PointeeComparison(const A *x, const A *y) {
```

Comment:
If this is being promoted to `idioms.h`, maybe the name should be normalized to a verb (which I should have done originally).